core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1080,6 +1088,10 @@ public function adjustDcaByType($dc)
 			case 'image':
 				$GLOBALS['TL_DCA']['tl_content']['fields']['imagemargin']['eval']['tl_class'] .= ' clr';
 				break;
+
+			case 'downloads':
```

Comment:
What about `case 'download'`? It the field not required there?

Reply:
Fixed in a22492738d2564b14414a141c86dfaff6fed5cec